internal/data/fixtures.go

```diff
@@ -730,14 +730,12 @@ func CreateMockImage(t *testing.T, width, height int, size ImageSize) image.Imag
 
 	var c color.Color
 	for x := 0; x < width; x++ {
-		if size == ImageSizeMedium {
-			c = mediumImageColor()
-		}
-
 		for y := 0; y < height; y++ {
 			switch size {
 			case ImageSizeSmall:
 				c = smallImageColor()
+			case ImageSizeMedium:
+				c = mediumImageColor()
```

Comment:
This is not the expected behavior. We should call the `mediumImageColor` function only for the `width`, otherwise, it will generate a large image.

Reply:
Thanks for catching that. This should fix it: https://github.com/stellar/stellar-disbursement-platform-backend/pull/163/commits/70559bb225d7a9264f8df3875c7baeb6a8851d69